.circleci/config.yml

```diff
@@ -90,21 +84,42 @@ jobs:
             bundle exec rake knapsack:rspec
             yarn test
             bundle exec slim-lint app/views
-
       - run:
           name: Code Climate Test Coverage
           command: |
             ./cc-test-reporter format-coverage -t simplecov -o "coverage/codeclimate.$CIRCLE_NODE_INDEX.json"
             aws s3 sync coverage/ "s3://login-gov-test-coverage/coverage/$CIRCLE_BUILD_NUM"
-
       # collect reports
       - store_test_results:
           path: /tmp/test-results
       - store_artifacts:
           path: /tmp/test-results
           destination: test-results
-
       - deploy:
           command: |
             aws s3 sync "s3://login-gov-test-coverage/coverage/$CIRCLE_BUILD_NUM" coverage/
             ./cc-test-reporter sum-coverage --output - --parts $CIRCLE_NODE_TOTAL coverage/codeclimate.*.json | ./cc-test-reporter upload-coverage --input -
+  build-container:
+    working_directory: ~/identity-idp
+    docker:
+      - image: circleci/ruby:2.6
+    steps:
+      - setup_remote_docker
+      - run: |
+          docker build -t logindotgov/idp:$CIRCLE_TAG -f production.Dockerfile .
+          echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin
+          docker push logindotgov/idp:$CIRCLE_TAG
```

Comment:
`CIRCLE_TAG` doesn't seem to be working here in this build, seems empty. Docs say it only has a value on tagged builds.

Reply:
That's correct, that build only runs when a release is cut.

Reply:
Fixed in dc936d3.